apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1798,60 +1837,64 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
-                                ".accessToken.fields.expiry.placeholder")
-                        }
-                        type="number"
-                        readOnly={ readOnly }
-                        min={ 1 }
-                        data-testid={ `${ testId }-access-token-expiry-time-input` }
-                    />
-                    <Hint>
-                        <Trans
-                            i18nKey={
-                                "console:develop.features.applications.forms.inboundOIDC.sections" +
-                                ".accessToken.fields.expiry.hint"
-                            }
-                        >
-                            Specify the validity period of the
-                            <Code withBackground>access_token</Code>
-                            in seconds.
-                        </Trans>
-                    </Hint>
-                </Grid.Column>
-            </Grid.Row>
-
+                                        ".accessToken.fields.expiry.labelForSPA")
+                                }
+                                required={ true }
+                                requiredErrorMessage={
+                                    t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                        ".accessToken.fields.expiry.validations.empty")
+                                }
+                                validation={ async (value: FormValue, validation: Validation) => {
+                                    if (!isValidExpiryTime(value.toString())) {
+                                        validation.isValid = false;
+                                        validation.errorMessages.push(
+                                            t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                                ".accessToken.fields.expiry.validations.invalid")
+                                        );
+                                    }
+                                } }
+                                value={
+                                    initialValues?.accessToken
+                                        ? initialValues.accessToken.userAccessTokenExpiryInSeconds.toString()
+                                        : metadata.defaultUserAccessTokenExpiryTime
+                                }
+                                placeholder={
+                                    t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                        ".accessToken.fields.expiry.placeholder")
+                                }
+                                type="number"
+                                readOnly={ readOnly }
+                                min={ 1 }
+                                data-testid={ `${ testId }-access-token-expiry-time-input` }
```

Comment:
Let's use component id instead

Reply:
data-testid was used to preserve the compatibility with tests